test/HandleHumanLocalMUnitTests.hs

```diff
@@ -51,6 +53,20 @@ handleHumanLocalMUnitTests = testGroup "handleHumanLocalMUnitTests"
                    in chooseItemProjectHuman testActorId triggerItems
       result <- executorCli testFn testCliStateWithItem
       showFailError (fromJust (fst result)) @?= "*aiming obstructed by terrain*"
+  , testCase "tutorialHints-msg-in-history-report" $ do
+      let testFn = tutorialHintMsgAdd CannotHarmYouInMelee
```

Comment:
Any particular reason you didn't just do `testFn = do modifySession ... {scurTutorial = True}; tutorialHintMsgAdd CannotHarmYouInMelee`? That would make the test longer, but with more information in one place.

Reply:
Thank you. That's easier to follow IMHO.